examples/slack-reaction-listener/functions/reactionAdded.ts

```diff
@@ -0,0 +1,24 @@
+import { GleeFunction } from "@asyncapi/glee"
+enum EVENT_TYPE {
+  EVENTS_API = "events_api",
+  CONNECTED = "hello"
+}
+const myFunction: GleeFunction = async ({ payload }) => {
+  const reaction = payload?.payload?.event?.reaction
+  if (!reaction) return
```

Comment:
I am getting this error `Cannot read properties of undefined (reading 'length')` when I am running the example and basically, this error is being thrown from https://github.com/asyncapi/glee/blob/163ff07844b8758e623498dcf2a272dcd16cd509/src/lib/util.ts#L93

Where the `data` parameter is undefined and I think `betterAjvErrors` is not handling it correctly. Also I double checked `reactionAdded` function is returning `undefined`.